content/maxwell3_fdem/emi_tutorial/induced_currents_body.rst

```diff
@@ -0,0 +1,94 @@
+   :align: center
+   :scale: 50%
+   :name: Concepts_3loops_only
+
+   Conceptual diagram for 3-loops system.
+
+Through the Tx (Loop1) we flow time-varying currents, then it generates time varying magnetic flux, which will exite the body (Loop2). Let :math:`\Phi^{p}` be the primary magnetic flux from the Tx. The body treats like and R-L circuilt.
+
+.. figure:: ./images/RLcircuit.png
+   :align: right
+   :scale: 60%
+   :name: RLcircuit
+
+   Conceptual diagram of RL circuit.
+
+Electrical impedance of the RL circuit can be written as
+
+.. math::
+    Z(\omega) = R + \imath \omega L,
+
+where R and L indicate resistance and inductance, respecitvely.
+Using Ohm's law we obtain
+
+.. math::
+    V(\omega) = I(\omega) Z(\omega).
+
+Voltage, can be expressed using :math:`\Phi^{p}` as
+
+.. math::
+    V = \mathcal{E} = -\frac{\partial \Phi^p}{\partial t} = -\imath \omega \Phi^p
+
+So,
+
+.. math::
+    I(\omega) = \frac{V(\omega)}{Z(\omega)} = \frac{-\imath\omega \Phi^p}{R+\imath \omega L}
+
+We can redefine this using time constant, :math:`\tau = \frac{L}{R}`:
+
+.. math::
+    I(\omega) = \frac{-\imath\omega \Phi^p}{R+\imath \omega L}
+
+or effectively,
+
+.. math::
+    I(\omega) = \frac{-\imath\omega \tau}{1+\imath \omega \tau} \frac{\Phi^p}{L},
+
+The quantity of :math:`\alpha = \omega \tau` is dimensionless, and
+
+.. math::
+    I(\omega) = \frac{-\imath \alpha}{1+\imath\alpha}\frac{\Phi^p}{L}.
+
+For small :math:`\alpha`, :math:`I(\omega) \simeq -\imath \alpha = -\imath \omega \tau`, so the current (and magnetic field) increases with frequency.
+
+Consider complex response function, :math:`Q` as
+
+.. math::
+    Q = \frac{\imath \alpha}{1+\imath\alpha}
+    = \frac{\imath \alpha(1-\imath\alpha)}{1+\alpha^2}
+    = \frac{\imath \alpha +\alpha^2}{1+\alpha^2}
+
+- for :math:`\alpha \ll 1\rightarrow` :math:`Q\simeq\imath\alpha` so :math:`I(\omega) \approx -\imath \alpha` (in the resitive limit)
+- for :math:`\alpha \gg 1\rightarrow` :math:`Q\simeq\imath\alpha` so :math:`I(\omega) \approx -1` (in the inductive limit)
+
+:numref:`CmplxResp` shows real and imaginary component of :math:`Q`.
+
+.. figure:: ./images/CmplxResp.png
+   :align: center
+   :scale: 80%
+   :name: CmplxResp
+
+   Complex response function.
+
+.. note::
+    For low frequencies the response is in the imaginary part, for high frequencies it will be in the real part.
+
+Phase of the current, :math:`\theta_I` can be written as
+
+.. math::
+    \theta_I = \theta_V - \theta_Z = -\frac{\pi}{2} - tan^{-1}\Big(\frac{\omega L}{R}\Big),
+    :label: PhaseI
+
+where :math:`\theta_z=\Re[z] / \Im[z]` and :math:`\Re[z]` and :math:`\Im[z]` are real and imaginary part of a complex value :math:`z`.
+
+Considering Faraday's law: :math:`V = -\imath\omega\Phi^p`, then recoginze factor of :math:`-\imath\omega`, which makes :math:`\frac{\pi}{2}` lag of phase in the induced current. And :math:`tan^{-1}(\frac{\omega L}{R})` lag of phase is induced by the Loop2.
```

Comment:
A couple things here: the grammar of the first sentence is a bit off, I would re-word to say something like
```
Considering Faraday's law: :math:`V = -\imath\omega\Phi^p`, the factor :math:`-\imath\omega` 
is responsible for the phase lag of :math:`\frac{\pi}{2}` in the induced current. 
```
for the second sentence, I think "lag of phase is induced" could be confusing (lots of inducing happening, currents, now phases!?). How about something like
```
An additional phase lag of :math:`\tan^{-1}(\frac{\omega L}{R}) is introduced by the presence of Loop2
```

Also: do you have a diagram you could add showing phase lag and what is meant by that ? (There is a lot happening in this statement, and a picture would do a lot to help the reader). I am thinking something along the line of showing a sine wave and then the relative shift at the target (if this makes no sense, we can chat about it!) 
<img width="561" alt="untitled-10 73 4 rgb gpu preview 2016-04-12 20-36-04" src="https://cloud.githubusercontent.com/assets/6361812/14482141/4f9ddb4c-00ee-11e6-8933-d5464f410312.png">